plotit.py

```diff
@@ -72,6 +74,9 @@ def sigint_handler(signum, frame):
 else:
     theme = 'default'
 
+if options.file_path:
+	file_path = str(options.file_path)
+
```

Comment:
@aditodkar I do not find where you have sent file_path param to the plot function. Am I missing something. Please point me out :)

Reply:
Check line 77 and line 23 @djmgit

Reply:
I have now added `file_path` param. Check line 109 & 113 but it is still not working am I missing something ?